lib/oli_web/components/delivery/content/content.ex

```diff
@@ -321,6 +377,22 @@ defmodule OliWeb.Components.Delivery.Content do
     end)
   end
 
+  defp maybe_filter_by_card(containers, nil), do: containers
+  defp maybe_filter_by_card(containers, ""), do: containers
+
+  defp maybe_filter_by_card(containers, selected_card_value) do
+    case selected_card_value do
+      "1" ->
```

Comment:
It would be easier for me to map and understand if we used a more explicit value instead of a string representation of a number. Something like the atoms you've already used above (`:high_progress_low_proficiency`, `zero_student_progress`) would make this code easier to understand IMO. You could also use pattern matching on this function definition instead of the case, but that's just a nitpick :)

You could end up with something like:
```elixir
defp maybe_filter_by_card(containers, :high_progress_low_proficiency) ...
defp maybe_filter_by_card(containers, :zero_student_progress) ...
defp maybe_filter_by_card(containers, _), do: containers
```

Reply:
I like your approach. Thanks